packages/plugins/src/cli/install/index.ts

```diff
@@ -9,28 +9,27 @@ import { sphinxUp } from './sphinxup'
 const installWithUpdate = async (
   spinner: ora.Ora,
   skipLibrary: boolean,
-  confirm: boolean
+  ci: boolean
 ) => {
-  if (!confirm) {
+  if (!ci) {
     await userConfirmation(
       'Would you like to install the Sphinx Foundry fork? This is required to use Sphinx and will replace your existing Foundry installation.\nConfirm? (y\\n):'
     )
-  }
+    const foundryupStatus = await spawnAsync('/bin/sh', [
```

Comment:
I've seen `/bin/bash` frequently, but I haven't seen `/bin/sh` before. Is this correct?